application/lib/model/KeyValuePair.dart

```diff
@@ -0,0 +1,5 @@
+class KeyValuePair<T1, T2> {
```

Comment:
Kunne du ikke bruge en map i Dart i stedet for?

Reply:
Nej, map er som udgangspunkt en liste og ikke et enkelt element. Det gør der svære at arbejde med.